static/app/components/feedback/useFetchFeedbackData.tsx

```diff
@@ -44,6 +44,7 @@ export default function useFetchFeedbackData(
     eventResult,
     issueData,
     issueResult,
+    hasReplay,
```

Comment:
Is this supposed to be here?

Reply:
looks like merge fails. i'll merge master in again and it's gone.